pyproject.toml

```diff
@@ -70,7 +70,7 @@ legacy_tox_ini = """
     requires=
         virtualenv>=20.13.2
         tox-gh-actions>=2.9.1
-    envlist = py38, py39, py310, pypy3
+    envlist = py38, py39, py310, py311, py312, py313, py314, py314t, pypy3
```

Comment:
You can remove `py314t` from here. Tox should use `py314` when gh actions setup `3.14t` matrix.

Reply:
```suggestion
    envlist = py38, py39, py310, py311, py312, py313, py314, pypy3
```